lib/gds_api/asset_manager.rb

```diff
@@ -127,6 +127,18 @@ def create_whitehall_asset(asset)
     post_multipart("#{base_url}/whitehall_assets", asset: asset)
   end
 
+  # Fetches a whitehall asset given the legacy URL path
```

Comment:
Minor: The docs for `create_whitehall_asset` capitalise "whitehall" when referring to the Whitehall app. It would be nice to be consistent in the two places where it's been used in the docs for the new method.